CHANGES.md

```diff
@@ -1,21 +1,8 @@
 Change log
 ==========
 
-1.1.0-rc2 (2015-01-29)
```

Comment:
Is there still a full history available somewhere after this has been removed? It's sometimes handy to be able to check when/in which version something was fixed/introduced.